src/ui/account_login_view.rs

```diff
@@ -130,7 +130,7 @@ fn login_textedit(manager: &mut LoginState) -> TextEdit {
     manager.get_login_textedit(|text| {
         egui::TextEdit::singleline(text)
             .hint_text(
-                RichText::new("Your key here...").text_style(NotedeckTextStyle::Body.text_style()),
+                RichText::new("Enter your public key (npub, nip05), or private key (nsec) here...").text_style(NotedeckTextStyle::Body.text_style()),
```

Comment:
do we support nip05 logins @kernelkind ?

Reply:
Yes